File: String/LengthOfLastWord.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        word_list = s.split()
        if word_list:
            last_word = word_list.pop()
            return len(last_word)
        return 0
        
            

File: String/LengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int l = s.length()-1;
        int len = 0;
        while (s[l]==' ' && l>=0){
            l--;
        }
        while(l>=0 && s[l]!= ' '){
            len++;
            l--;
        }
        return len;
    }
};
